### ISSUE TITLE
[Bug] Wildcard not working in Lerna exec

### ISSUE THREAD

**Issue by Siilwyn** [lerna/lerna#352]
So I would like to symlink all CLI tools that are commonly used and stumbled upon the fact that the `exec` command doesn't expand wildcards.

`lerna exec -- ln -rs ../../node_modules/.bin/* ./node_modules/.bin/`
Expected result: symlinks of all files in the first passed directory to be in the second passed directory.
Result: a symlink named '_' linking to '../../../../node_modules/.bin/_' in the second passed directory.

Current workaround is using a script file like:

``` js
#!/usr/bin/env node
'use strict';

const exec = require('child_process').exec;

exec('ln -rs ../../node_modules/.bin/* ./node_modules/.bin/', function (error, stdout, stderr) {
  if (error) {
    console.error(`exec error: ${error}`);
    return;
  }
  console.log(stdout);
  console.log(stderr);
});
```

And running it with `lerna exec -- node ../../symlink.js`

**Comment by thejameskyle** [comment]
This is a limitation of our use of `spawn` since it's just using a string. Maybe we can work around it like this: http://stackoverflow.com/questions/11717281/wildcards-in-child-process-spawn

**Comment by thejameskyle** [comment]
Relavent code: https://github.com/lerna/lerna/blob/master/src/ChildProcessUtilities.js#L45-L56

**Comment by thejameskyle** [comment]
Just published a module to handle this: https://github.com/thejameskyle/cross-glob-spawn

I haven't yet tested it out on windows, but I think it will work since node uses it

**Comment by Siilwyn** [comment]
Wow hyperfast response @thejameskyle! Do you want to use `cross-glob-spawn` in the Lerna codebase?

**Comment by thejameskyle** [comment]
I'll use it once I can test on Windows

**Comment by thejameskyle** [comment]
Actually, we already have windows being tested in CI. I'll just see what happens.

**Comment by evocateur** [comment]
I believe this was fixed in https://github.com/lerna/lerna/commit/bed393e6b843b1f1559868d24d30dca8ab327440 (#761), released in [v2.0.0-rc.2](https://github.com/lerna/lerna/releases/tag/v2.0.0-rc.2).
